### ISSUE TITLE
Local storage of orientation and ephemerides

### ISSUE THREAD

**Issue by mcbeth** [astropy/astropy#7513]
My use of astropy.coordinates is usually on a machine without a network connection.  I can consistently point at a local finals2000A.all without too much trouble, but the best I've accomplished is pointing at the de430.bsp as a file URL.  It feels wrong to have the process go out and copy the files off into another location before it runs, and I'd prefer both files be used in place so the file system could be read-only/non-existent.

**Comment by mhvk** [comment]
Yes, good idea. For someone aiming to implement this: the place to loop is `coordinates/solar_system`, probably best to change `_get_kernel`.
